Traceback (most recent call last):
  File "bad4numba.py", line 2, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
